low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -312,7 +312,7 @@ def validate(self):
         pass
 
     def process(self):
-        args = ' '.join(self._args)
+        args = ' '.join(self._args.args)
```

Comment:
@mariyappanp Have you tested the SSPL support bundle and Manifest support bundle file generation after this code change?

Reply:
Ok confirm this plz

Reply:
Yes, tested.